dashboard/src/reducers/namespace.ts

```diff
@@ -24,7 +24,7 @@ const namespaceReducer = (
     case getType(actions.namespace.setNamespace):
       return { ...state, current: action.namespace };
     case LOCATION_CHANGE:
-      const pathname = action.payload.pathname;
+      const pathname = action.payload.location.pathname;
```

Comment:
it's weird that this is the only breaking change, isn't it? I know it's fine but it's a good opportunity to add unit tests :)

Reply:
It seems so, the main change I can see is this one and the use of `connectRouter(history)(rootReducer)` instead of making the user add a router key in the combineReducers. I looked at the code and it does exactly that.

I have added some unit tests on the locationChange case. Testing the whole router might be too much for this patch and I am not sure how much it overlaps with future integration tests. I have done some manual testing and the app seems to work as expected.